docs/en/latest/http3.md

```diff
@@ -0,0 +1,113 @@
+---
+title: HTTP3 protocol
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+`APISIX` supports HTTP3 protocol. Currently, it only supports opening HTTP/3 connection between downstream and APISIX.
```

Comment:
```suggestion
[HTTP/3](https://en.wikipedia.org/wiki/HTTP/3) is the third major version of the Hypertext Transfer Protocol (HTTP). Unlike its predecessors which rely on TCP, HTTP/3 is based on [QUIC (Quick UDP Internet Connections) protocol](https://en.wikipedia.org/wiki/QUIC). It brings several benefits that collectively result in reduced latency and improved performance:

 * enabling seamless transition between different network connections, such as switching from Wi-Fi to mobile data.
 * eliminating head-of-line blocking, so that a lost packet does not block all streams.
 * negotiate TLS versions at the same time as the TLS handshakes, allowing for faster connections.
 * providing encryption by default, ensuring that all data transmitted over an HTTP/3 connection is protected and confidential.
 * providing zero round-trip time (0-RTT) when communicating with servers that clients already established connections to.

 APISIX currently supports HTTP/3 connections between downstream clients and APISIX. HTTP/3 connections with upstream services is not yet supported.

 :::caution

 This feature has not been tested at scale and therefore, is not recommended for production use.

 :::
```